examples/using-listeners/index.js

```diff
@@ -0,0 +1,44 @@
+const Tram = require('../../tram-one')
+const app = new Tram()
+const html = Tram.html()
+
+const countActions = {
+  init: () => parseInt(localStorage.count, 10) ? parseInt(localStorage.count, 10) : 0,
+  up: (count) => count + 1,
+  down: (count) => count - 1
+}
+
+const clicker = (store, actions) => {
+  const up = () => {
+    actions.up()
+  }
+  const down = () => {
+    actions.down()
+  }
+
+  return html`
+    <div>
+      You can use Tram-One's
+      <a href="https://github.com/JRJurman/hover-engine">Hover-Engine</a>
+      to listen for state change, and debug or save whatever the current state is.
+      <br />
+      <div>
+        Current Value: ${store.counter}
+      </div>
+      <button onclick=${up}>
+        UP
+      </button>
+      <button onclick=${down}>
+        DOWN
+      </button>
+    </div>
+  `
+}
+
+app.addListener(store => {
+  console.log('Current Value', store.counter)
```

Comment:
nit: I get that a console log might be useful, but maybe we could show this in a way where they don't have to open debugging tools. maybe just set the text inside a div to the last state change?
Honestly not that big of a deal tho... this is fine fore an example.

Reply:
Eh, I don't want to change the view though, because honestly, if that's your intention, you should be doing that view manipulation with Tram-One.

Reply:
Fair point :+1: